python/GafferMLUI/ImageToTensorUI.py

```diff
@@ -103,6 +105,18 @@
 
 		},
 
+		"tensorDataType" : {
+
+			"description" :
+			"""
+			The data type to use for the output tensor.
```

Comment:
Is there anything else we can add here? Perhaps "Should be chosen to match the requirements of the model used in the Inference node."?